autoload/preview_markdown.vim

```diff
@@ -47,7 +47,12 @@ function! preview_markdown#preview() abort
     let opt['term_finish'] = 'close'
   endif
 
-  call term_start(parser, opt)
+  if has('nvim')
+    vnew
```

Comment:
If `get(g:, 'preview_markdown_vertical', 0)` return 1, then should execute `vnew`